docs/BulkExport.md

```diff
@@ -0,0 +1,37 @@
+# Bulk Export
+
+This feature allows data from the FHIR server to be exported. More detail can be found in the [spec](https://github.com/HL7/bulk-data/blob/master/spec/export/index.md).
+
+The feature is currently under preview and is turned off by default. To enable the feature, update the `FhirServer:Operations:Export:Enabled` setting to be true.
+
+## Specifying destination
+
+One customization that we have done in addition to the spec is _requiring_ the destination information to be supplied. This is a breaking change from the spec. We are planning to support default destination to maintain compatibility with spec.
+
+The destination type can be specified by using `_destinationType` query parameter and any destination specific settings can be specified as base64 encoded string by using `_destinationConnectionSettings` query parameter.
+
+Currently, we only support Azure Blob storage as the destination. This can be specified with `_destinationType=azure-block-blob` and `_destinationConnectionSettings=[base64 encoding of the connection string]`.
```

Comment:
I am not sure about the terminology here. But do we want to specifically call out the connection string should be for the storage account itself (and not, say, for a container/blob). Or is it already implied?

Reply:
I think it's implied. There is also an example of what the connection string looks like in the spec.